### PR TITLE
Updates the questions.yml for rancher

### PR DESCRIPTION
**Pull request by 0sewa0**
# Description
Updates the `questions.yml` to use all the relevant fields from the `values.yaml`

## How can this be tested?
TODO

## Checklist
- [x] PR is labeled accordingly

### CODE REVIEW

File: config/helm/chart/default/questions.yml [hunk 1]
```diff
@@ -3,29 +3,164 @@ categories:
...
+    description: "Installs the Custom Resource Definitions for the Dynakube. This is recommended if you haven't installed it manually yet. Default: true"
+    default: true
+    type: boolean
+    group: "Global Configuration"
+
   - variable: operator.image
-    label: "Set a custom image for operator"
-    description: "Optional: Set a custom image for operator. Defaults to docker.io/dynatrace/dynatrace-operator"
+    label: "Set a custom image for operator components"
+    description: "Set a custom image for operator. Defaults to docker.io/dynatrace/dynatrace-operator"
```
Comment (chrismuellner) [inline config/helm/chart/default/questions.yml:16]: defaults to tenant registry

Comment (chrismuellner) [inline config/helm/chart/default/questions.yml:16]: why remove the optional?

File: config/helm/chart/default/values.yaml [hunk 2]
```diff
@@ -52,6 +52,4 @@ csidriver:
 
 securityContextConstraints:
   enabled: true # Only applicable for Openshift
-  
-activeGate:
```
Comment (chrismuellner) [inline config/helm/chart/default/values.yaml:56]: is this not used anymore?

**Review by chrismuellner — CHANGES_REQUESTED** [review]

File: config/helm/chart/default/values.yaml (hunk 2 above)
Comment (0sewa0) [reply config/helm/chart/default/values.yaml:56]: yep, it used to set a feature flag on the CR

File: config/helm/chart/default/questions.yml (hunk 1 above)
Comment (0sewa0) [reply config/helm/chart/default/questions.yml:16]: everything is optional, felt redundant 
should I add `Optional` to every field ?

Comment (chrismuellner) [reply config/helm/chart/default/questions.yml:16]: you're right, not needed

Comment (0sewa0) [reply config/helm/chart/default/questions.yml:16]: I could update the `app-readme.md` so it say that the configuration fields are optional
or have the section names have the word `Optional`

Comment (0sewa0) [reply config/helm/chart/default/questions.yml:16]: there is an operator image in the tenant registry ?

Comment (chrismuellner) [reply config/helm/chart/default/questions.yml:16]: whoops

**Review by chrismuellner — APPROVED** [review]

**Pull request merged**